validator/build.gradle

```diff
@@ -20,14 +20,14 @@ dependencies {
     implementation "org.alvearie.hri.flink:hri-flink-pipeline-core:${pipelineCoreVersion}"
     implementation "commons-io:commons-io:2.4" // not sure why this isn't getting picked up from hri-flink-pipeline-core
     implementation "org.apache.flink:flink-connector-kafka_${scalaBinaryVersion}:${flinkVersion}" // not sure why this isn't getting picked up from hri-flink-pipeline-core
-    implementation "com.ibm.fhir:fhir-model:4.7.1"
+    implementation "com.ibm.fhir:fhir-model:4.7.1" 
 
     //Test dependencies here:
     testImplementation "org.scalactic:scalactic_${scalaBinaryVersion}:${scalaTestVersion}"
     testImplementation "org.scalatest:scalatest_${scalaBinaryVersion}:${scalaTestVersion}"
 
-    testRuntimeOnly  "log4j:log4j:${log4jVersion}"
-    testRuntimeOnly  "org.slf4j:slf4j-log4j12:${slf4jVersion}"
+    testRuntimeOnly  "org.apache.logging.log4j:log4j-core:${log4jVersion}"
+    testRuntimeOnly "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
```

Comment:
Just curious why the group and artifacts changed. Were they incorrect before?

Reply:
> Just curious why the group and artifacts changed. Were they incorrect before?

The log4J project hanged the group/artifact naming conventions in the switch from log4j 1.x to log4j 2.x. They moved all of their libs to a new org name (`org.apache.logging.log4j`).  Not sure why.